pkg/volume/csi/csi_client.go

```diff
@@ -158,12 +158,12 @@ func (c *csiDriverClient) NodeGetInfo(ctx context.Context) (
 	maxVolumePerNode int64,
 	accessibleTopology map[string]string,
 	err error) {
-	klog.V(4).Info(log("calling NodeGetInfo rpc"))
+	klog.V(4).InfoS(log("calling NodeGetInfo rpc"))
```

Comment:
Note the log() method currently adds "kubernetes.io/csi: " to the beginning of the log message. If I understand the structured logging project correctly, that might be better expressed as a structured key-value "plugin=kubernetes.io/csi".

I'm not sure though of a nice way to inject this sort of information short of using a custom InfoS method. Maybe you have better ideas?

Reply:
In my opinion, "kubernetes.io/csi" can be regarded as a label of the log, and there are too many callers, maybe we keep the status quo?

Reply:
Ok, I think that's fine to keep the status quo. As structured logging evolves we may discover a more reasonable solution to systematically adding annotations.